src/lib-components/overlays/Tooltip.vue

```diff
@@ -36,9 +37,9 @@ const hoverPopover = (e: MouseEvent, open: boolean): void => {
 
 <template>
   <Popover :position="position" :as="as">
-    <template #button="{ open, close }">
+    <template #button="{ open, close }: { open: boolean, close: () => void }">
```

Comment:
This is just a type annotation for the slot props, which are not currently available via inference with HeadlessUI components.